### PR TITLE
adjusted configmap template

### PR DESCRIPTION
**Pull request by adeniyistephen**

### CODE REVIEW

File: standard-app/templates/configs/configmap.yaml
Comment (kirillplis) [inline standard-app/templates/configs/configmap.yaml:1]: Victor suggested writing a snippet, please work on it https://cloudkite.slack.com/archives/C2CT6QDMH/p1696334112540229

File: standard-app/example.values.yaml [hunk 1]
```diff
@@ -1,6 +1,7 @@
 image: us-central1-docker.pkg.dev/cloudkite-infra-ops/cloudkite-docker-images/example-image
 tag: example_tag
 
+enableConfigMap: true
```
Comment (kirillplis) [inline standard-app/example.values.yaml:4]: It makes no sense to use `enableConfigMap` key when you have consecutive conditions checks afterwards
```
{{- if .Values.enableConfigMap }}
{{- if $appConfig.volumes }}
{{- range $appConfig.volumes }}
{{- if .type.configMap }}
```

**Review by kirillplis — CHANGES_REQUESTED** [review]
@adeniyistephen It's unclear to me what you are trying to achieve with these changes and why. 
If you're doing a refactoring Victor suggested - you should follow his recommendations. If you're doing something else - I don't know anything about it. Please add a description.

**Comment by adeniyistephen** [comment]
I'm deleting this PR, I'm working on the Template, I just wanted to get a test on the argocd scripts. I don't want it all to look different.

**Pull request closed without merging**